src/python/gudhi/representations/vector_methods.py

```diff
@@ -606,16 +606,16 @@ class Atol(BaseEstimator, TransformerMixin):
     >>> c = np.array([[3, 2, -1], [1, 2, -1]])
     >>> atol_vectoriser = Atol(quantiser=KMeans(n_clusters=2, random_state=202006))
     >>> atol_vectoriser.fit(X=[a, b, c]).centers
```

Comment:
What happens if we append `#doctest: +SKIP` to this line?

Reply:
Not enough. cf. [failing tests before my modification](https://dev.azure.com/GUDHI/gudhi-devel/_build/results?buildId=808&view=logs&j=5264e576-3c6f-51f6-f055-fab409685f20&t=24836e8d-1ec0-5c42-ef10-bcd7ccf7525e)
```txt
Document: representations
-------------------------
**********************************************************************
File "../../../build/src/python/gudhi/representations/vector_methods.py", line ?, in default
    atol_vectoriser(a)
Expected:
    array([1.18168665, 0.42375966])
Got:
    array([0.42375966, 1.18168665])
**********************************************************************
File "../../../build/src/python/gudhi/representations/vector_methods.py", line ?, in default
Failed example:
    atol_vectoriser(c)
Expected:
    array([0.02062512, 1.25157463])
Got:
    array([1.25157463, 0.02062512])
**********************************************************************
File "../../../build/src/python/gudhi/representations/vector_methods.py", line ?, in default
Failed example:
    atol_vectoriser.transform(X=[a, b, c])
Expected:
    array([[1.18168665, 0.42375966],
           [0.29861028, 1.06330156],
           [0.02062512, 1.25157463]])
Got:
    array([[0.42375966, 1.18168665],
           [1.06330156, 0.29861028],
           [1.25157463, 0.02062512]])
**********************************************************************
```

Reply:
Skip doctests on the 4 lines that are failing on 2a29df7c

Reply:
Yes of course it would need to be done for each line, but the point was to see if it allows reverting the patch that comments out the output, so it would look normal again in the generated doc.

Reply:
If I say:
```python
    >>> atol_vectoriser.fit(X=[a, b, c]).centers #doctest: +SKIP
    array([[ 2.        ,  0.66666667,  3.33333333],
           [ 2.6       ,  2.8       , -0.4       ]])
```
It fails saying:
```txt
Failed example:
    atol_vectoriser.fit(X=[a, b, c]).centers
Expected nothing
Got:
    array([[ 2.        ,  0.66666667,  3.33333333],
           [ 2.6       ,  2.8       , -0.4       ]])
```

Reply:
Ah :frowning_face: Maybe with the `#doctest: +SKIP` also on those 2 lines...
I guess anything is ok to quickly get the release out, but we should try to get something pretty soon afterwards.